models/Comment.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const CommentSchema = new Schema(
+  {
+    userId: { type: Schema.Types.ObjectId, ref: "User" },
+    businessId: { type: Schema.Types.ObjectId, ref: "Business" },
+    dishId: { type: Schema.Types.ObjectId, ref: "Dish" },
+    userName: String,
+    title: String,
+    content: String,
```

Comment:
I added to our google docs another field for `Comment` - Source IP.
We can save the IP of the client that sent the comment (based on the request), and then translate it to a country later.
Anyway, forget what we will do with it in the future (out of scope), just add a `sourceIp` field here